static/js/lib/queries/courses_test.js

```diff
@@ -17,6 +17,7 @@ describe("Courses API", () => {
   beforeEach(() => {
     sandbox = sinon.createSandbox()
     results = R.times(makeCourse, 10)
+    results.sort()
```

Comment:
what's this for here? just curious

Reply:
One of these tests randomly failed once, I think because the courses came back in a different order than usual.

Reply:
ah, that is probably related to the course id collision issue I mentioned above I think

Reply:
I just removed this line because it was trying to fix the wrong problem